nemo/collections/tts/data/datalayers.py

```diff
@@ -207,3 +217,244 @@ def __getitem__(self, index):
 
     def __len__(self):
         return self.samples.shape[0] // self.n_segments
+
+
+"""
+    This dataset was implmented using code from: https://github.com/Sytronik/deep-griffinlim-iteration/blob/master/dataset.py 
+
+# MIT License

```

Comment:
Let's move this license to the top of the file, and move `This dataset was implemented using code from: https://github.com/Sytronik/deep-griffinlim-iteration/blob/master/dataset.py` inside the docstring of the NoisySpecsDataset